src/panel/category/CategoryPanel.jsx

```diff
@@ -149,19 +150,27 @@ export default class CategoryPanel extends React.Component {
       if (hasError) {
         panelContent = <CategoryPanelError zoomIn={zoomIn} />;
       } else {
-        panelContent = <PoiItemList
-          pois={pois}
-          selectPoi={this.selectPoi}
-          highlightMarker={this.highlightPoiMarker}
-        />;
+        panelContent =
+        <>
+          <PoiItemList
+            pois={pois}
+            selectPoi={this.selectPoi}
+            highlightMarker={this.highlightPoiMarker}
+            dataSource={dataSource}
+          />
+          {dataSource === sources.pagesjaunes &&
+          <div className="category__panel__pj">
+            {_('Results in partnership with PagesJaunes', 'categories')}
+          </div>}
+        </>;
       }
     }
 
     return <Panel
+      white
       resizable
-      renderHeader={<CategoryPanelHeader dataSource={dataSource} loading={initialLoading} />}
-      minimizedTitle={_('Show results', 'categories')}
-      className="category__panel"
+      minimizedTitle={_('Unfold to show the results', 'categories')}
+      className={classnames('category__panel', { 'panel--pj': dataSource === 'pages_jaunes' })}
```

Comment:
in the lines above, you use `dataSource === sources.pagesjaunes` for the same test. This one should be the same.